bluebrain/sysconfig/bluebrain5/packages.yaml

```
@@ -8,6 +8,9 @@ packages:
     variants: +gas+gold+ld+plugins~libiberty
   boost:
     variants: +filesystem+pic+serialization+test
+  caliper:
+    # see https://github.com/LLNL/Caliper/issues/529
+    version: [2.9.1]
```

```suggestion
    require: "@2.9.1"
```
for a "hard" requirement